src/view/MenuBar.java

```diff
@@ -0,0 +1,86 @@
+package view;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+
+/**
+ * Created by Marcela Barrionuevo on 16/05/2016.
+ */
+public class MenuBar extends JMenuBar implements ActionListener {
+
+    private JMenu files;
+    private JMenu mark;
+    private JMenu commands;
+    private JMenu show;
+    private JMenu configuration;
+    private JMenu help;
+
+    private JMenuItem filesChangeAttrib;
+    private JMenuItem filesCompareContent;
+    private JMenuItem filesProperties;
+    private JMenuItem filesQuit;
+
+    public MenuBar() {
+        initComponents();
+    }
+
+    private void initComponents() {
+
+        files = new JMenu("Files");
+        files.setMnemonic(KeyEvent.VK_F);
+        mark = new JMenu("Mark");
+        mark.setMnemonic(KeyEvent.VK_M);
+        commands = new JMenu("Commands");
+        commands.setMnemonic(KeyEvent.VK_C);
+        show = new JMenu("Show");
+        show.setMnemonic(KeyEvent.VK_W);
+        configuration = new JMenu("Configuration");
+        configuration.setMnemonic(KeyEvent.VK_O);
+        help = new JMenu("Help");
+        help.setMnemonic(KeyEvent.VK_H);
+
+        filesChangeAttrib = new JMenuItem("Change Attributes", new ImageIcon(MenuBar.class.getResource("../image/iconAttributes.gif")));
+        filesChangeAttrib.addActionListener(this);
+        filesChangeAttrib.setMnemonic(KeyEvent.VK_A);
+        filesChangeAttrib.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_A, ActionEvent.ALT_MASK));
+        filesChangeAttrib.getAccessibleContext().setAccessibleDescription("Change attributes in selected folders/files");
+        filesCompareContent = new JMenuItem("Compare By Content", new ImageIcon(MenuBar.class.getResource("../image/iconCompare.png")));
```

Comment:
try to use empty lines to separate logically the code, like after filesChangeAttrib add an empty line